test/file-size/format-file-size.js

```diff
@@ -0,0 +1,38 @@
+import format from '../../src/file-size/format';
+
+let chai = require('chai'),
+	expect = chai.expect;
+
+chai.should();
+
+describe('file-size', () => {
+	describe('format-file-size', () => {
+		[
+			{byteNum:0, expect:'0 bytes'},
+			{byteNum:-1, expect:'-1 bytes'},
```

Comment:
Actually shelved till Monday, the current ideas are keep it like it is, or on a negative show '0 bytes'